### PR TITLE
Added: FFA domain

### PR DESCRIPTION
**Pull request by GoliathLabs**
Good day,
I have added the FFA domain to our site config. Since I'm still planning some projects over in Augsburg and the FFA community is not really active anymore, I would like to get FFA up and running again with the support of Freifunk Munich.

Steps needed:
- [x] Allocate IP ranges in Netbox
- [x] Add interfaces and IPs to each gateway in Netbox
- [x] Add the domain to config_context `sites` in Netbox
- [X]  Add the domain to `wgkex.yaml` EDIT by @GoliathLabs: I've created a PR for this: https://github.com/freifunkMUC/site-ffm/pull/151
- [x] Test the rollout on one Gateway with `test=true` (salt-run jobs.lookup_jid 20211020104928365427)
- [x]  Roll it out on one gateway and try to connect with a node
- [x]  Roll it out on all gateways
- [X]  Add the new domain to meshviewer EDIT by @GoliathLabs: I've created a PR for this: https://github.com/freifunkMUC/meshviewer/pull/5

EDIT by @krombel; copy TODO-List to first comment to enable github to track progress

### CODE REVIEW

**Review by goligo — CHANGES_REQUESTED** [review]
next_node IP addresses need to be adapted to the new prefixes as well. 

I am not sure whether we should use "muenchen.freifunk.net" SSIDs for Augsburg, even if they run on our infrastructure. Can't we just use "augsburg.freifunk.net" instead?

**Comment by awlx** [comment]
Until now every domain which runs under muenchen.freifunk.net gets their suffix. That's because of some reasons:
1. It's clear FFMUC runs this domain
2. No meshing with old nodes in the original domain
3. It's consistent with /gauting and /freising etc

File: domains/ffmuc_augsburg.conf [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  },
+
+  dns = {
+    servers = {
+      '2001:678:e68:ff00::3',
+      '2001:678:e68:ff00::4',
+    },
+  },
+
+  next_node = {
```
Comment (awlx) [inline domains/ffmuc_augsburg.conf:41]: Please fix the nextnode addresses as well

**Review by awlx — CHANGES_REQUESTED** [review]

File: domains/ffmuc_augsburg.conf (hunk 1 above)
Comment (GoliathLabs) [reply domains/ffmuc_augsburg.conf:41]: Done

**Comment by awlx** [comment]
For the admins @lqb, @goligo and @krombel that also means that we need to allocate those ranges in netbox and roll the changes out on the Gateways.

**Comment by awlx** [comment]
Steps needed:
- [ ] Allocate IP ranges in Netbox
- [ ] Add interfaces and IPs to each gateway in Netbox
- [ ] Add the domain to config_context `sites` in Netbox
- [X]  Add the domain to `wgkex.yaml` EDIT by @GoliathLabs: I've created a PR for this: https://github.com/freifunkMUC/site-ffm/pull/151
- [ ] Test the rollout on one Gateway with `test=true`
- [ ]  Roll it out on one gateway and try to connect with a node
- [ ]  Roll it out on all gateways
- [X]  Add the new domain to meshviewer EDIT by @GoliathLabs: I've created a PR for this: https://github.com/freifunkMUC/meshviewer/pull/5

**Comment by GoliathLabs** [comment]
@awlx we also need to add the new domain to our meshviewer config

**Comment by awlx** [comment]
@GoliathLabs the process would be even easier if `wgkex.yaml` and meshviewer config get autogenerated by salt. Maybe a good first salt PR for you.

**Review by awlx — APPROVED** [review]

**Comment by awlx** [comment]
Even tho I approved this, it shouldn't be merged until all other steps are fulfilled.

**Review by goligo — APPROVED** [review]

**Pull request merged**